geniesp/bpc_redcap_export_mapping.py

```diff
@@ -707,14 +707,32 @@ def create_fixed_timeline_files(self, timeline_infodf, timeline_type):
             "used": used_entities,
         }
 
+    
+    def get_mg_synid(self, synid_folder: str, file_name: str) -> str:
```

Comment:
Might be more efficient here is you just build a dictionary:

{"filename": "synapseid"}

Then you don't need to call `getChildren` everytime you want to get a file.

Reply:
For instance:

```
{synid_child['name']: synid_child['id'] for synid_child in synid_children}
```

Reply:
Agreed, would be more efficient.  But just for the sake of clarity, I'd rather not create, store, and pass, another custom dictionary around whose contents are not immediately apparent to the user.  The `get_mg_synid` is only called 5 times and doesn't add hugely to the runtime so I think it's worth the tradeoff in transparency.

Reply:
Hmmm.  I'm ok with the implementation.  But I think having something like:
```
cna_synid = mg_synid_map["data_CNA.txt"]
```

Is similar to what you have - I guess the transparency is that the code itself tells you that you're getting the synapse id of a certain file, instead of having to explain what the `mg_synid_map` variable is.  Although, someone external to GENIE would still have to go to the function `get_mg_synid` to understand what that function is doing.